WordPress/Classes/Models/Notifications/Notification.swift

```diff
@@ -256,7 +256,7 @@ extension Notification {
         }
 
         guard let timestamp = timestamp, let timestampAsDate = Date.dateWithISO8601String(timestamp) else {
-            DDLogSwift.logError("Error: couldn't parse date [\(self.timestamp)] for notification with id [\(notificationId)]")
+            DDLogSwift.logError("Error: couldn't parse date [\(String(describing: self.timestamp))] for notification with id [\(notificationId)]")
```

Comment:
As an alternative, I'd prefer to use `.debugDescription` instead of `String(describing:)`